flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/utils/UserDefinedAggFunctions.java

```diff
@@ -59,7 +59,7 @@ public boolean requiresOver() {
 	/**
 	 * Accumulator for WeightedAvg.
 	 */
-	public static class WeightedAvgAccum extends Tuple2<Long, Integer> {
```

Comment:
Yes, you are right. Either we change the type or we touch the code and use `acc.f0` instead of `acc.sum` and `acc.f1` instead of `acc.count`.

Let's go with your approach.